CHANGELOG.rst

```diff
@@ -4,6 +4,29 @@ Changelog
 This CHANGELOG documents only key changes between versions. For a full description 
 of all changes see https://github.com/ACCESS-NRI/access-nri-intake-catalog/releases
 
+v0.1.5
+------
+
+21/11/2024
+
+- Switch to a date-based catalog versioning schema
```

Comment:
Maybe additionally note catalog version no longer bound to package version?

Reply:
We're going to have to update the changelog for the release anyway, perhaps we make that point there?